### ISSUE TITLE
constructor does not work with non-bzip2 packages

### ISSUE THREAD

**Issue by newville** [conda/constructor#258]
It appears that `constructor` generates shell scripts that cannot unpack themselves if some of the packages are not `bzip2`.   

`constructor -v` now reports packages with a `.conda` extension instead of a `.tar.bz2` extension.    This seems to be related to conda-build 3.18.5, though I'm not 100% sure.

Is there any easy work-around?

**Comment by AXGKl** [comment]
confirmed :-(

**Comment by dinya** [comment]
Temporary recipe works for me:

1. Create venv:

    ```sh
    conda create --name "build_venv" "python==3.7.*" "conda<4.5.13" "conda-build < 3.18.3" "constructor==2.3.*"
    ```

2. Activate and build the dist there:

    ```sh
    conda activate build_venv
    ```

See also https://github.com/conda/conda/issues/8857

**Comment by nehaljwani** [comment]
xref: https://github.com/conda/constructor/issues/86#issuecomment-510924775

**Comment by msarahan** [comment]
fixed in constructor 3.0
